src/sedcli_main.c

```diff
@@ -329,10 +329,30 @@ static int get_password(struct sed_key *pwd)
 	return 0;
 }
 
+static void sed_status_ret(int status)
+{
+	const char *sed_status = NULL;
+
+	sed_status = sed_error_text(status);
+
+	if (sed_status == NULL && status != 0) {
+		if (status == -EINVAL) {
+			sedcli_printf(LOG_ERR, "Invalid parameter\n");
+		} else if (status == -ENODEV) {
+			sedcli_printf(LOG_ERR, "Couldn't determine device state\n");
+		} else if (status == -ENOMEM) {
+			sedcli_printf(LOG_ERR, "No memory\n");
+		} else {
+			sedcli_printf(LOG_ERR, "Unknown error\n");
+		}
+	 } else {
+		 sedcli_printf(LOG_INFO, "%s\n", sed_status);
```

Comment:
please fix indentation

Reply:
done

Reply:
> done

Compilation erros:

sedcli_main.c: In function ‘handle_ownership’:
sedcli_main.c:387:2: error: implicit declaration of function ‘sed_status_ret’ [-Werror=implicit-function-declaration]
  sed_status_ret(ret);
  ^
sedcli_main.c: At top level:
cc1: error: unrecognized command line option "-Wno-address-of-packed-member" [-Werror]
cc1: all warnings being treated as errors

Reply:
sorry its the end of the week effect :(
done